src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2721,10 +2744,15 @@ namespace winrt::TerminalApp::implementation
             if (bothAltsPressed)
             {
                 std::tie(connection, debugConnection) = OpenDebugTapConnection(connection);
+                // Debug Tap connections aren't really restartable. The
```

Comment:
so what happens if you _do_ restart one?

Reply:
You can restart the underlying connection, but not the debugtap-ness of it. The debug spew just stops, but the real pane gets restarted.